### ISSUE TITLE
sasl.mechanism not populated for kafkabridge.spec.authentication.type=scram-sha-256

### ISSUE THREAD

**Issue by yanxuehe** [strimzi/strimzi-kafka-bridge#947]
org.apache.kafka.common.config.ConfigException: Invalid value null for configuration sasl.mechanism: When the security.protocol configuration enables SASL, mechanism must be non-null and non-empty string.

```yaml
apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaBridge
metadata:
  name: kafka-bridge-0
spec:
  replicas: 1
  image:
  bootstrapServers: kafka:9092
  authentication:
    type: scram-sha-256
    username: user1
    passwordSecret:
      secretName: kafka-user-passwords
      password: system-user-password
  http:
    port: 8080
```

**Comment by yanxuehe** [comment]
[PR](https://github.com/strimzi/strimzi-kafka-bridge/pull/948) raised.

**Comment by scholzj** [comment]
Fixed in #948
